src/v1/atm/index.js

```diff
@@ -0,0 +1,6 @@
+import updateAtmService from './atm-update-service';
+import atmRouter from './atm-router';
+
+updateAtmService.updateAtmsDataInDB(); // TODO: Use cron to start update on same time
```

Comment:
it's an async function, so if it throws an error it will be silent, either to .catch() here, or try {} catch {} in function body